docs/diagrams/PoolSequenceDiagram.puml

```diff
@@ -0,0 +1,90 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":PoolCommandParser" as PoolCommandParser LOGIC_COLOR
+participant "p:PoolCommand" as PoolCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+participant "toAdd:Pool" as Pool MODEL_COLOR
+end box
+
+[-> LogicManager : execute(command)
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand(command)
+activate AddressBookParser
+
+create PoolCommandParser
```

Comment:
It might be a minor thing but in the textbook I don't think it wasn't mentioned that the constructor call can be ignored? Maybe you might have to put the PoolCommandParser() method call here just to be safe? Might be a very small issue as Prof might have said that this could be ignored during tutorial but I'm not sure.